BricksAndHearts.UnitTests/ControllerTests/Landlord/LandlordControllerTestsBase.cs

```diff
@@ -1,12 +1,28 @@
-﻿using BricksAndHearts.Controllers;
-using BricksAndHearts.Services;
+﻿using System;
 using BricksAndHearts.ViewModels;
-using FakeItEasy;
 
 namespace BricksAndHearts.UnitTests.ControllerTests.Landlord;
 
 public class LandlordControllerTestsBase : ControllerTestsBase
 {
-    public static readonly IPropertyService propertyService = A.Fake<IPropertyService>();
-    protected readonly LandlordController _underTest = new(null!, null!, new LandlordService(null!), propertyService);
+    protected PropertyViewModel CreateExamplePropertyViewModel()

```

Comment:
Think this might conflict with Max's work on BNH-34

Reply:
I rebased very recently (right before I left yesterday afternoon) so it should be fine.